scripts/core/filters/index.ts

```diff
@@ -1,9 +1,7 @@
+import _ from 'lodash';
 import {stripHtmlTags} from '../utils';
 
 export default angular.module('superdesk.core.filters', [])
-    .filter('any', () => function(data, key) {
```

Comment:
Why did you remove this?

Reply:
It was using `_.any` which is deprecated and I search for usages of the filter "any" in superdesk and couldn't find one